profile/nodejs.sh

```diff
@@ -1,11 +1,13 @@
 calculate_concurrency() {
   MEMORY_AVAILABLE=${MEMORY_AVAILABLE-$(detect_memory 512)}
   WEB_MEMORY=${WEB_MEMORY-512}
-  WEB_CONCURRENCY=${WEB_CONCURRENCY-$((MEMORY_AVAILABLE/WEB_MEMORY))}
-  if (( WEB_CONCURRENCY < 1 )); then
-    WEB_CONCURRENCY=1
+  if [ -z "$WEB_CONCURRENCY" ]; then
+    WEB_CONCURRENCY=${WEB_CONCURRENCY-$((MEMORY_AVAILABLE/WEB_MEMORY))}
+    if (( WEB_CONCURRENCY < 1 )); then
+      WEB_CONCURRENCY=1
+    fi
+    WEB_CONCURRENCY="0$WEB_CONCURRENCY"
```

Comment:
IMO it'd be better if this function `echo`d the result, instead of modifying an outside variable (and then the caller does `export WEB_CONCURRENCY=$(calculate_concurrency)`